src/common.h

```diff
@@ -996,6 +996,11 @@ static inline int psf_lrint (double x)
 ** Other helper functions.
 */
 
+static inline float clampf (float x, float min, float max)
```

Comment:
@arthurt , this function is pretty useful.

Reply:
1) It operates in the float domain (slow)
2) It's a pretty trivial function
3) It's different from how clipping is handled elsewhere in the project.